observing/cli.py

```diff
@@ -63,6 +63,20 @@ def create_sdf(sdffile, n_obs, sess_mode, beam_num, obs_mode, obs_start, obs_dur
                    obs_dur=obs_dur, ra=ra, dec=dec, obj_name=obj_name, int_time=int_time)
 
 
+@cli.command()
+@click.argument('mjd')
+@click.argument('sdffile')
+def submit_command(mjd, command):
+    """ Submit a command to be added to schedule at time mjd.
+    Command should be python code that can be evaluated, complete with imports.
+    E.g., "import time; time.sleep(10)" will sleep for 10 seconds.
+    """
+
+    # TODO: sanitize and/or test that command is value when evaluated
+
+    ls.put_dict('/cmd/observing/submitsdf', {'mjd': mjd, 'command': command, 'mode': 'buffer'})
```

Comment:
um. arbitrary code execution? Should still limit "command" to a subset of strings that the scheduler recognize though?

Reply:
Agreed. I'll sanitize it by limiting it to settings changes for now.